src/components/common/CofirmDialog/ConfirmButton.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { button } from './ConfirmDialog.css';
+
+interface ConfirmButtonProps {
+  title: string;
+  onClick?: () => void;
+}
+export default function ConfirmButton({ title, onClick }: ConfirmButtonProps) {
+  const handleButtonClick = () => {
+    onClick?.();
+  };
+
+  return (
+    <button className={button} onClick={handleButtonClick}>
```

Comment:
```suggestion
    <button className={button} onClick={onClick}>
```
 (마이너) 요렇게 줄일 수 있을거같아요